server/.env

```
@@ -1,2 +1,2 @@
-TEAM_NAME="Shums"
+TEAM_NAME="Shums Kassm"
```

Spelling mistake - should be Kassam
```suggestion
TEAM_NAME="Shums Kassam"
```